lib/graph.ts

```diff
@@ -351,7 +376,7 @@ class GraphImpl implements Graph, Graph.Transformer {
             throw new Error(`node outputs should not be initialized`);
           }
           if (data._from !== nodeIndex) {
-            throw new Error(`from property of the Value object doesn't match index of Node being processed`);
+            // throw new Error(`from property of the Value object doesn't match index of Node being processed`);
```

Comment:
There's a line I forgot to uncomment. It was used to suppress the error by the graph transformation. Now it can be safely uncommented.